ocean_provider/utils/util.py

```diff
@@ -37,6 +38,10 @@ def get_request_data(request):
     return request.args if request.args else request.json
 
 
+def msg_hash(message: str):
```

Comment:
am I confusing this with aquarius? I am sure such a function already exists. It has to... or there are cases where the same logic is used, where we need to use this function instead.

either way, please add a test for this function specifically in test_utils.

Reply:
I didn't see anything similar.